docs/source/about/guides/guides.rst

```diff
@@ -0,0 +1,10 @@
+Guides
+======
+
+Collection of guides written by the community! 
+
+.. toctree::
+   :maxdepth: 2
+   
+   app_examples
+   linux
```

Comment:
I wonder if we should have sub categories / folders.

i.e.

- general/app_examples
- linux/headless_ssh

Just because cramming another guide into this Linux one is going to make it an even bigger monster.

Reply:
Yes lol, I agree.

Reply:
We talked over discord, but wanted to update this thread. I was thinking of having generic / misc guides to be flat under `about/guides` to limit the number of mouse clicks for the user.
```
about/guides
├── app_examples.rst
├── guides.rst
└── linux
    └── headless_ssh.rst

   ```
![2023-09-07-172941_280x190_scrot](https://github.com/LizardByte/Sunshine/assets/16567905/92f0467a-4fa5-49c7-9e85-1f452fbbafd8)

Reply:
This is fine for now. If it gets to be too much, we can re-think it later.